pkg/commands/addresource.go

```diff
@@ -70,9 +70,23 @@ func (o *addResourceOptions) Complete(cmd *cobra.Command, args []string) error {
 
 // RunAddResource runs addResource command (do real work).
 func (o *addResourceOptions) RunAddResource(fsys fs.FileSystem) error {
-	if !fsys.Exists(o.resourceFilePath) {
-		return errors.New(o.resourceFilePath + " does not exist")
+	var resources []string
+
+	for _, pattern := range o.resourceFilePath {
+		files, err := fsys.Glob(pattern)
+		if err != nil {
+			return err
+		}
+		if len(files) == 0 {
+			return errors.New(pattern + " does not exist")
```

Comment:
s/does not exist/has no match/

Also: This feels more like a case of log.Infof(...) rather than an error return.